packages/farms/src/utils.ts

```diff
@@ -65,18 +61,19 @@ const formatStableUniversalFarmToSerializedFarm = (
     stableLpFee: stablePair.stableLpFee,
     stableLpFeeRateOfTotalFee: stablePair.stableLpFeeRateOfTotalFee,
     infoStableSwapAddress: stablePair.infoStableSwapAddress,
-    bCakeWrapperAddress: bCakeConfig?.bCakeWrapperAddress,
+    bCakeWrapperAddress,
     chainId,
     version: 2,
   }
 }
 
 const formatV2UniversalFarmToSerializedFarm = (farm: UniversalFarmConfigV2): LegacyClassicFarmConfig => {
-  const { chainId, pid, lpAddress, token0, token1 } = farm
+  const { chainId, pid, bCakeWrapperAddress, lpAddress, token0, token1 } = farm
   return {
-    pid,
+    pid: pid ?? 0,
```

Comment:
<img width="556" alt="image" src="https://github.com/user-attachments/assets/b0fffa07-e24f-4278-9959-ce00ddf56847">
we have config with a pid=0, not sure is there would be cause some issues if fallback pid to 0

Reply:
done, will ignore all farms that without pid